virttest/libvirt_version.py

```diff
@@ -24,16 +25,22 @@ def version_compare(major, minor, update):
     :param major: Major version to compare against
     :param minor: Minor version to compare against
     :param update: Update value to compare against
+    :param session: Shell session on remote host
     :return: True if running version is greater than or
                   equal to the input libvirt version
     """
     global LIBVIRT_LIB_VERSION
 
+    func = process.system_output
+    if session:
+        func = session.cmd_output
+        LIBVIRT_LIB_VERSION_ORG, LIBVIRT_LIB_VERSION = LIBVIRT_LIB_VERSION, 0
```

Comment:
I don't think those assignment is related to version_compare() itself. What version_compare does here is just to give true or false based on your input parameter, therefore please remove those unrelated variables assignment here

Reply:
@chunfuwen Updated.